src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * This DeleteCommand class will delete a task from the list of tasks.
+ */
+public class DeleteCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete";
+    private final int taskIndex;
+
+    /**
+     * Constructor for DeleteCommand which provides an index to delete.
+     *
+     * @param taskIndex 0-based index.
```

Comment:
I like how you specified the index being 0-based here. It's always confusing switching between 0-based and 1-based! 👍